tests/test_kubernetes_tools.py

```diff
@@ -280,11 +283,11 @@ def test_get_cmd_list(self):
             service="kurupt",
             instance="fm",
             cluster="brentford",
-            config_dict={"cmd": ["/bin/echo", "hi"]},
+            config_dict={"cmd": "/bin/echo hi"},
             branch_dict=None,
             soa_dir="/nail/blah",
         )
-        assert deployment.get_cmd() == ["/bin/echo", "hi"]
+        assert deployment.get_cmd() == ["sh", "-c", "/bin/echo hi"]
```

Comment:
what's this doing here?

Reply:
Oh, I see your comment about mypy in tests/.

Most things in tests/ aren't really meant to be mypy-tested; I think sometimes mypy is a little too strict with tests, which are big on duck typing.

Reply:
I think you added some of these files to the list of what mypy should check :) https://github.com/Yelp/paasta/blame/master/tox.ini